src/v2/schema/query.js

```diff
@@ -22,6 +22,7 @@ type Query {
 
   # Search for resources.
   search(input: [SearchInput]): [SearchResult]
+  messages: [Message]
```

Comment:
This comment documents the API.
```suggestion

  # Additional information about the search service status. This is similar to errors, but without implying that there was a problem.
  messages: [Message]
```